### PR TITLE
fix: Revert #1164 (Clones re-ordering)

### PR DESCRIPTION
**Pull request by DafyddLlyr**
## What does this PR do?

 - Reverts logical changes made in https://github.com/theopensystemslab/planx-new/pull/1164
 - Removes redundant `act()` and `waitForExpect()` calls which were throwing "not wrapped in act(...)" errors in test (see https://github.com/theopensystemslab/planx-new/pull/1141 for context)

Pizza passed along to George and Jenna for testing.

### CODE REVIEW

**Review by jessicamcinchak — APPROVED** [review]
thanks for rolling this one back! reminder that secure file upload is on staging so can't be easily/immediately linearly deployed to prod tho!

**Comment by DafyddLlyr** [comment]
Thanks @jessicamcinchak  @gunar  👍 I was aware of that and should have pointed at `production` for this PR - that's now updated.

Waiting on testing feedback to indicate if this fixes the issue (seems pretty likely) before I merge.

**Comment by DafyddLlyr** [comment]
An update from Jenna who's been testing on this Pizza - 

> I've tested and it's still the same. The outcome of 'permitted development' or 'planning permission required' is still being skipped. I've asked [@Garry Hildersley](https://opendigitalplanning.slack.com/team/U0323DB0ZD5) to test as well and if he has the same outcome to then message [@Georgina Murray](https://opendigitalplanning.slack.com/team/U02UP5PLF17) with the breakdown of what's happening so we can bottom this.

**Pull request merged**
